testing/trino-benchto-benchmarks/src/main/resources/sql/presto/tpcds/q39.sql

```diff
@@ -29,18 +29,18 @@ WITH
       GROUP BY "w_warehouse_name", "w_warehouse_sk", "i_item_sk", "d_moy"
    )  foo
    WHERE ((CASE "mean" WHEN 0 THEN 0 ELSE ("stdev" / "mean") END) > 1)
-) 
+)
 SELECT
   "inv1"."w_warehouse_sk"
 , "inv1"."i_item_sk"
 , "inv1"."d_moy"
 , "inv1"."mean"
-, "inv1"."cov"
+, CAST("inv1"."cov" AS DECIMAL(30, 10))
```

Comment:
this requires comment

Reply:
Comments are included in commit message - do you want them in code?

Reply:
> Comments are included in commit message - do you want them in code?

Yes.